cdisc_rule_tester/services/config_service.py

```diff
@@ -0,0 +1,40 @@
+import os
```

Comment:
This is no longer needed as well

Reply:
is logging_service still needed?

Reply:
You're right, I don't think we need logging_service as well